src/lightning_app/utilities/cli_helpers.py

```diff
@@ -35,3 +41,55 @@ def _format_input_env_variables(env_list: tuple) -> Dict[str, str]:
 
         env_vars_dict[var_name] = value
     return env_vars_dict
+
+
+def _is_url(id: Optional[str]) -> bool:
```

Comment:
```suggestion
def _is_url(url: Optional[str]) -> bool:
```
probably should be moved to utilities